external/mainnet/UniswapV2Router.json

```diff
@@ -1,976 +1,4 @@
 {
```

Comment:
Instead of deleting this file or `KeepToken.json`, I propose we rename `external/mainnet` to `external/mainnet-v1` and keep the files unchanged there. We can then have `external/mainnet` for v2 or `externa/mainnet-v2`. IMO second option is cleaner.

Reply:
Done here, but why do we need to keep `mainnet-v1`? In this PR we are removing a bunch of deployment scripts related to v1, and I take that some of the code will also be changed/removed in the next PRs. Maybe we should remove it and just leave `mainnet`?

Reply:
For `@keep-network/keep-core` we decided not to remove contracts but to archive them into [`solidity-v1` directory](https://github.com/keep-network/keep-core/tree/main/solidity-v1). The reasoning was that these contracts are deployed to mainnet and still in use. The same situation is here except that it would be harder to archive them, given we are going to use part of the contracts for v2.  That's why I [suggested](https://github.com/keep-network/coverage-pools/pull/203#discussion_r1052028944) marking contracts that are used only in v1 in a `@dev` comment.

Now, we have a question what to do with v1 mainnet deployment artifacts. If we are not deleting contracts but archiving them, I think it makes sense to keep artifacts as well. That's just my personal preference so I can see us relying only on the version control but... this wouldn't be consistent with the decision of archiving contracts IMO.

cc @nkuba for thoughts here